src/synapse/azext_synapse/_help.py

```diff
@@ -154,122 +163,165 @@
+        --workspace-name testsynapseworkspace --resource-group rg
 """
 
 helps['synapse sql pool show'] = """
 type: command
 short-summary: Get a SQL pool.
-long-summary: Get a SQL pool with SQL pool name.
 examples:
   - name: Get SQL pool.
     text: |-
-        az synapse sql pool show --name sqlpoolcli1 --resource-group rg --workspace-name \\
-        testsynapseworkspace
+        az synapse sql pool show --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg
 """
 
 helps['synapse sql pool list'] = """
 type: command
 short-summary: List all SQL pools.
-long-summary: List all SQL pools under a specific workspace.
 examples:
   - name: List SQL pools.
     text: |-
-        az synapse sql pool list --resource-group rg \\
-        --workspace-name testsynapseworkspace
+        az synapse sql pool list --workspace-name testsynapseworkspace --resource-group rg
+"""
+
+helps['synapse sql pool update'] = """
+type: command
+short-summary: Update SQL pool.
+examples:
+  - name: Update SQL pool.
+    text: |-
+        az synapse sql pool update --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg \\
+        --tags key1=value1
 """
 
 helps['synapse sql pool pause'] = """
 type: command
 short-summary: Pause a SQL pool.
-long-summary: Pause a SQL pool with SQL pool name.
 examples:
   - name: Pause SQL pool.
     text: |-
-        az synapse sql pool pause --name sqlpoolcli1 --resource-group rg --workspace-name \\
-        testsynapseworkspace
+        az synapse sql pool pause --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg
 """
 
 helps['synapse sql pool resume'] = """
 type: command
 short-summary: Resume a SQL pool.
-long-summary: Resume a SQL pool with SQL pool name.
 examples:
   - name: Resume SQL pool.
     text: |-
-        az synapse sql pool resume --name sqlpoolcli1 --resource-group rg --workspace-name \\
-        testsynapseworkspace
+        az synapse sql pool resume --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg
 """
 
 helps['synapse sql pool delete'] = """
 type: command
 short-summary: Delete a SQL pool.
-long-summary: Delete a SQL pool with SQL pool name.
 examples:
   - name: Delete SQL pool.
     text: |-
-        az synapse sql pool delete --name sqlpoolcli1 --resource-group rg --workspace-name \\
-        testsynapseworkspace
+        az synapse sql pool delete --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg
 """
 
 helps['synapse sql pool wait'] = """
 type: command
 short-summary: Place the CLI in a waiting state until a condition of a SQL pool is met.
 """
 
+helps['synapse workspace firewall-rule'] = """
+type: group
+short-summary:  Manage a workspace's firewall rules.
+"""
+
+helps['synapse workspace firewall-rule create'] = """
+type: command
+short-summary: Create a firewall Rule.
+examples:
+  - name: Create a firewall Rule.

```

Comment:
>Rule [](start = 28, length = 4)

rule

Reply:
fixed

---
In reply to: [418048104](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418048104) [](ancestors = 418048104)